stable/datadog/README.md

```diff
@@ -53,6 +53,7 @@ The following tables lists the configurable parameters of the Datadog chart and
 | `resources.limits.cpu`      | CPU resource limits                | 512Mi                                     |
 | `resources.requests.memory` | Memory resource requests           | 100m                                      |
 | `resources.limits.memory`   | Memory resource limits             | 256m                                      |
+| `daemonset.tolerations`     | List of node taints to tolerate    | `nil`                                     |
```

Comment:
Can do! Do you think it would be worth it to add a `daemonset.podAnnotations` value to accommodate 1.5 clusters?